python-toolbox/tests/management/test_notebook.py

```diff
@@ -40,7 +40,7 @@ def test_notebook(system_mocked, sys_mocked):
     spark_conf = '/opt/spark/conf'
     system_mocked.return_value = 1
 
-    notebook(ctx, port, enable_security, spark_conf, allow_root)
+    notebook(ctx, port, '/', enable_security, spark_conf, allow_root)
```

Comment:
I think it is better to still create a variable called `base_url` instead of directly put a `'/'` in the code.